lib/elixir/pages/unicode-syntax.md

```diff
@@ -34,12 +34,12 @@ Elixir does not allow the use of ZWJ or ZWNJ in identifiers and therefore does n
 
 ### Atoms
 
-Unquoted unicode identifier-based atoms in Elixir follow the identifier rule above with the following modifications:
+Unicode atoms in Elixir follow the identifier rule above with the following modifications:
```

Comment:
I like this clarification. But my intention with that change ( I was also not bought by the idea of over-specifying it), was to not to rely on the note for mentioning that is this ruly only apply for identifier-based atoms.
I guess a good compromise could be just to mention "Unquoted atoms in Elixir".... at the opening of the parapgrah. WDTY @josevalim?

Reply:
unquote is ambiguous in Elixir (it can be about macros) - so unless we clearly specify what unquoted means, then a reference to it most likely won't help - because it probably makes sense only if you know what it means.

Reply:
You are right. I have submitted a PR addressing this.